src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCosts = 40;
+
+  if (days >= 7) {
+    return rentCosts * days - 50;
+  } else if (days >= 3) {
+    return rentCosts * days - 20;
+  } else {
+    return rentCosts * days;
+  }
```

Comment:
add 50 and 20 to the variables and check your code according to [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md)